colosseum/shapes.py

```diff
@@ -0,0 +1,38 @@
+class Rect:
+    """Representation of a rectangular shape."""
+
+    def __init__(self, top, right, left, bottom):
+        self._top = top
+        self._right = right
+        self._left = left
+        self._bottom = bottom
+
+    def __eq__(self, other):
+        return other.__class__ == self.__class__ and other.to_tuple() == self.to_tuple()
```

Comment:
Is this the only reason to have to_tuple? The extra function call isn't free, and the previous check has confirmed both objects are `Rect`... is there any reason we don't just confirm equality of the four attributes?

Reply:
Hmmm nope, the only reason was this, yes!

Fixed